frontend/base/src/main/resources/GitMacheteBundle.properties

```diff
@@ -124,11 +123,16 @@ action.GitMachete.BaseSlideInBranchBelowAction.description.disabled.no-parent=Sl
 action.GitMachete.BaseSlideInBranchBelowAction.dialog.create-new-branch.title=Create New Branch From {0}
 action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.checkbox.reattach=Reattach children
 action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.label.branch-name=Branch name:
-action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.label.parent=Parent Branch:
+action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.label.parent=Parent branch:
 action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.ok-button=Slide In
 action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.title=Slide In Branch Below
-action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.error.slide-in-under-itself=Cannot slide in the branch under itself
-action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.error.slide-in-under-its-descendant=Cannot slide in the branch under any of its descendants
+action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.no-branch=No branches
+action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.no-branch-selected=The merge branch should be selected.
+action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.error.slide-in-under-itself=Cannot slide in the branch under itself.
+action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.placeholder=Specify branch
+action.GitMachete.BaseSlideInBranchBelowAction.dialog.slide-in.progress=Loading unmerged branches\u2026
```

Comment:
What's that unicode stuff?

Reply:
[horizontal epilepsis](https://en.wikipedia.org/wiki/Ellipsis) 👀

Reply:
Wow okay... is that visibly different from just plain `...`?

Reply:
<img width="89" alt="image" src="https://user-images.githubusercontent.com/19799111/176115737-5fe8ba38-1011-4375-a1c0-af44b154194e.png">

The `\u2026` is more compact. Some text editors (e.g. for latex) often suggest using the special character for 3 dots instead of 3 dots - I guess that the visual is one thing, and grammar is the other (a usual dot used just after a dot is taken as an error). 

I would like to change 3 dots to \u2026 everywhere. WDYT @PawelLipski ?

Reply:
Wow okay... consider adding a check in `com.virtuslab.gitmachete.resourcebundles.junit.GitMacheteBundlePropertiesTestSuite`